### PR TITLE
Feat/reusable agent plugin

### LINKED ISSUE #1244: feat: add reusable PowerContext skills and Agent Plugin packaging
### Feature description

Add a reusable PowerContext Agent Plugin package for agents that support Agent Plugin skills and MCP configuration.

The package should provide:

- reusable PowerContext skills that describe how agents should use PowerContext Memory and Handoff;
- MCP configuration for the existing PowerContext Streamable HTTP MCP endpoint;
- Agent Plugin packaging so compatible agents can load the same package.

This creates a shared integration package for future agent integrations.

### Problem and proposed solution

PowerContext exposes Memory and Handoff operations through its existing MCP endpoint. Agents that support Agent Plugin skills and MCP configuration should be able to reuse the same PowerContext guidance and MCP setup instead of defining similar instructions separately for each agent.

The proposed solution is to add a reusable Agent Plugin package containing PowerContext skills and MCP configuration.

The package should reuse the existing PowerContext MCP endpoint and tool surface. It should not duplicate Runtime or Memory behavior. Memory search, writes, revisions, Handoff behavior, and persistence remain owned by the PowerContext Server and existing MCP tools.

The work can be split into two PRs:

1. define reusable PowerContext skills;
2. add reusable MCP configuration and Agent Plugin packaging.

### Alternatives considered

_No response_

### Additional context

This follows the discussion in #1213.

The maintainer suggested defining reusable PowerContext skills and MCP configuration, then packaging them as an Agent Plugin. This can be split into two PRs and may support multiple agents through the same package.

### Are you willing to contribute to this feature?

- [x] Yes, I am willing to contribute code, docs, or design feedback.

### PR DESCRIPTION
**Pull request by curryxjh**
# Which issue or RFC does this PR close?

Closes #1244.

# Rationale for this change

This change adds a reusable PowerContext Agent Plugin package so compatible agents can load PowerContext skills and MCP configuration without relying on agent-specific integration files.

# What changes are included in this PR?

- Add `integrations/agent-plugin/powercontext/` with portable Agent Plugin metadata and MCP configuration.
- Add a reusable `project-context` skill that documents PowerContext Memory and Handoff workflows through MCP.
- Document how to configure the Agent Plugin package in both English and Chinese docs.
- Add contract tests for the plugin manifest, MCP configuration, reusable skill content, and README boundaries.

# Are there any user-facing changes?

Yes. Users now have a documented reusable Agent Plugin package for compatible agents.

There are no breaking API changes and no persisted format changes.

# How was this change tested?

- `UV_CACHE_DIR=/private/tmp/powercontext-uv-cache uv run pytest tests/agent_plugin/test_contract.py`

# AI usage statement

Used OpenAI Codex to help implement the Agent Plugin packaging, documentation, tests.

### CODE REVIEW

File: integrations/agent-plugin/powercontext/skills/project-context/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+## Hand Off Current Work
+
+Use Handoff when work must move to another task, session, model, or compatible
+agent.
+
+1. Call `capture_content_source` with a concise account of the current state
+   and a unique `source_id`. Include the objective, verified progress, blockers,
+   and next action that the receiver needs.
+2. Call `activate_handoff` with that Source as `boundary_source`. Add any other
```
Comment (Teingi) [inline integrations/agent-plugin/powercontext/skills/project-context/SKILL.md]: The default `powercontext server run` configuration has no generation model, so this path fails at `activate_handoff` with `503 handoff_generation_unavailable`. Please use the model-free `handoff_current_work` flow here (then `continue_handoff`, `acknowledge_handoff`, and `record_task_outcome`) and add a default-configuration contract test.

File: docs/en/docs/how-to/configure-agent-plugin.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+title: Configure Agent Plugin
+description: Load reusable PowerContext skills and MCP configuration in compatible agents.
+---
+
+# Configure Agent Plugin
+
+PowerContext provides a portable Agent Plugin package for agents that can load
+Agent Plugin skills and MCP configuration.
+
+The package lives in the repository at:
```
Comment (Teingi) [inline docs/en/docs/how-to/configure-agent-plugin.md]: A normal PowerContext wheel only ships `src/powercontext`, so this repository path is not present after installation. The guide also never shows how a compatible client obtains or loads the directory. Please provide a shipped artifact or setup command and at least one verified host-loading procedure; schema validation alone does not make the plugin usable.

File: docs/en/docs/index.md [hunk 3]
```diff
@@ -1,51 +1,26 @@
 ---
-template: docs-overview.html
```
Comment (Teingi) [inline docs/en/docs/index.md:2]: This conflict resolution removes the existing `docs-overview` homepage and its Pi, OpenCode, Handoff, Memory, Experience, and Skill entry points in both locales. Please preserve the current structured homepage and add the Agent Plugin entry to it and to `zensical.toml` instead of replacing the page.

**Review by Teingi — COMMENTED** [review]
Three inline findings from exact-head validation.

File: docs/en/docs/how-to/configure-agent-plugin.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
```
Comment (Teingi) [inline docs/en/docs/how-to/configure-agent-plugin.md]: The VS Code loading steps are valid once the repository has already been cloned, but the acquisition gap remains: the standard wheel only ships `src/powercontext`, and this guide still provides no clone/download/install step that creates the configured directory. Please add an obtainable artifact or an explicit setup command before asking users to register this path.

File: docs/en/docs/index.md [hunk 5]
```diff
@@ -18,9 +18,6 @@ overview:
         - title: Continue in Pi
           description: Open project context in Pi with the native package.
           href: en/docs/how-to/configure-pi/
-        - title: Continue in OpenClaw
```
Comment (Teingi) [inline docs/en/docs/index.md:21]: This now drops the OpenClaw card that is present on the current base, and the English homepage still does not add the Agent Plugin card that was added to the Chinese homepage. Please preserve the OpenClaw entry and add the matching Agent Plugin entry here so the two locales stay aligned.

**Review by PsiACE — APPROVED** [review]
totally lgtm
